### PR TITLE
Fix source formatting issues

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

File: liferay-gradle/source-formatter.properties [hunk 1]
```diff
@@ -1,4 +1,10 @@
 line.length.excludes=\
 	blade.doclib.resourcecommand.override/src/main/java/com/liferay/blade/samples/doclib/resourcecommand/override/CustomMVCResourceCommand.java,\
 	blade.hook.resourcebundle/src/main/java/com/liferay/blade/samples/hook/resourcebundle/ResourceBundleLoaderComponent.java,\
-	blade.portlet.controlpanel/src/main/java/com/liferay/blade/samples/portlet/controlpanel/ControlPanelDemo.java
\ No newline at end of file
+	blade.portlet.controlpanel/src/main/java/com/liferay/blade/samples/portlet/controlpanel/ControlPanelDemo.java
```
Comment (lawrence-lee) [inline liferay-gradle/source-formatter.properties]: @gamerson I added these exclusions because we didn't author these classes.

Comment (gamerson) [reply liferay-gradle/source-formatter.properties]: Lets go ahead and fix that class instead of ignoring it.

**Comment by lawrence-lee** [comment]
Hi @bruinen, we have copyrighted your original classes to Apache 2.0.

**Comment by lawrence-lee** [comment]
jenkins retest

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/liferay/liferay-blade-samples/pull/145. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**